package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-config-formidable",
   "description": "A set of default eslint configurations from formidable",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "authors": "eric.baer@formidable.com, ryan.roemer@formidable.com, kyle.cesmat@formidable.com",
```

Comment:
I agree totally, but I might have missed your meaning behind why the npm cli is dangerous? My method for tagging/publishing is usually:
- `npm version ( major | minor | patch )` (this is where the package.json is changed)
- `npm publish`
- `git push && git push --tags`

Is that not the desired method?